packages/api/src/external/fhir/document/index.ts

```diff
@@ -327,7 +325,8 @@ function extractAuthorOrganization(contained: Resource[]): Reference<FhirOrganiz
   const org = contained.find(r => r.resourceType === "Organization");
   if (org) {
     return {
-      reference: `#${org.id}`,
+      // https://www.hl7.org/fhir/R4/references-definitions.html#Reference.reference
+      reference: `Organization/${org.id}`,
```

Comment:
Correct way to reference resources available on the FHIR server

Reply:
Hmm this doesn't seem right:

- the Organization in contained will be the Organization coming from CW
- we've determined that the Organization from CW will *not* be in the FHIR server outside of this resource
- so, the `#id` would be the correct reference in this case, no?